kerngen/pisa_generators/rotate.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Module containing implementation for rotate kernel."""
+
+from dataclasses import dataclass
+
+from high_parser.pisa_operations import PIsaOp, Comment
+from high_parser import KernelContext, HighOp, Polys, KeyPolys
+
+from .basic import Add, mixed_to_pisa_ops
+from .relin import KeyMul, DigitDecompExtend, extract_last_part_polys
```

Comment:
```suggestion
from .basic import Add, KeyMul, mixed_to_pisa_ops, extract_last_part_polys
from .decomp import DigitDecompExtend
```

Reply:
Hi @christopherngutierrez, remember that KeyMul, extract_last_part_polys, and DigitDecompExtend were moved out from relin.py by the refactoring introduced by PR #47